benefits/in_person/views.py

```diff
@@ -30,6 +32,7 @@ def eligibility(request):
             flow_id = form.cleaned_data.get("flow")
             flow = models.EnrollmentFlow.objects.get(id=flow_id)
             session.update(request, flow=flow)
+            eligibility_analytics.started_eligibility(request, flow, enrollment_method=models.EnrollmentMethods.IN_PERSON)
```

Comment:
I would think we also want to send the `selected enrollment flow` event here, right? For consistency with the `digital` side.

Reply:
Yep, good idea. Added it in 159594176a75282614d75a815a6409af8501f07a